great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -333,8 +334,8 @@ def _reconcile_batch_requests_in_builders(
         Args:
             batch_request: Data provided at runtime used to hydrate nested builder attributes
         """
-        if isinstance(batch_request, dict):
-            batch_request = get_batch_request_from_acceptable_arguments(**batch_request)
+        if not isinstance(batch_request, dict):
+            batch_request = get_batch_request_as_dict(batch_request)
```

Comment:
@cdkini This may be useful, but I am not sure if this is sufficient.  Please see the earlier comments.  Thanks!